codegen/cmd/injection-gen/generators/reconciler_reconciler.go

```diff
@@ -426,20 +430,22 @@ func (r *reconcilerImpl) Reconcile(ctx {{.contextContext|raw}}, key string) erro
 		reconcileEvent = rof.ObserveFinalizeKind(ctx, resource)
 	}
 
-	// Synchronize the status.
-	if {{.equalitySemantic|raw}}.DeepEqual(original.Status, resource.Status) {
-		// If we didn't change anything then don't call updateStatus.
-		// This is important because the copy we loaded from the injectionInformer's
-		// cache may be stale and we don't want to overwrite a prior update
-		// to status with this stale state.
-	} else if !isLeader {
-		logger.Warn("Saw status changes when we aren't the leader!")
-		// TODO: Consider logging the diff at Debug?
-	} else if err = r.updateStatus(original, resource); err != nil {
-		logger.Warnw("Failed to update resource status", zap.Error(err))
-		r.Recorder.Eventf(resource, {{.corev1EventTypeWarning|raw}}, "UpdateFailed",
-			"Failed to update status for %q: %v", resource.Name, err)
-		return err
+	if r.skipStatusUpdates {
```

Comment:
bool is flipped (commenting again on visible file)

Reply:
ugh, rushing...